deps/GraphBLAS/README.md

```diff
@@ -218,3 +218,4 @@ BSD-2-clause license.
 Intel (also Apache 2.0), and the cpu_featurer/ndk_compat component (c)
 The Android Open Source Project (BSD-2-clause)
 
+CLEAR CACHE
```

Comment:
can't we force CI clear cache via the CI platform ?

Reply:
Unfortunately no or you change the key or like in our case change the README file

Reply:
fuggedaboutit
https://circleci.com/docs/2.0/caching/#clearing-cache